seek-n-major.json

```diff
@@ -0,0 +1,135 @@
+      "commitMessageTopic": "{{depName}}",
+      "matchManagers": ["buildkite", "gomod", "npm", "nvm"]
+    },
+    {
+      "matchManagers": ["npm"],
+      "matchPackageNames": ["@types/node"],
+      "matchUpdateTypes": ["major"],
+      "enabled": false
+    },
+    {
+      "automerge": true,
+      "commitMessageExtra": "",
+      "groupName": "npm definitely typed",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@types/"],
+      "prPriority": 99,
+      "recreateClosed": true,
+      "matchUpdateTypes": ["major"],
+      "schedule": "before 3am on Tuesday"
+    },
+    {
+      "commitMessageExtra": "",
+      "depTypeList": ["devDependencies"],
+      "excludePackageNames": ["braid-design-system", "sku", "skuba"],
+      "excludePackagePatterns": ["^@?seek", "seek$", "^@types/"],
+      "groupName": "npm dev dependencies",
+      "matchManagers": ["npm"],
+      "recreateClosed": true,
+      "matchUpdateTypes": ["major"],
+      "schedule": "after 3am and before 6am on Tuesday"
+    },
+    {
+      "commitMessageExtra": "",
+      "depTypeList": ["peerDependencies"],
+      "excludePackageNames": ["braid-design-system", "sku", "skuba"],
+      "excludePackagePatterns": ["^@?seek", "seek$", "^@types/"],
+      "groupName": "npm peer dependencies",
+      "matchManagers": ["npm"],
+      "recreateClosed": true,
+      "matchUpdateTypes": ["major"],
+      "schedule": "after 3am and before 6am on Tuesday"
+    },
+    {
+      "commitMessageExtra": "",
+      "excludePackagePatterns": ["^seek-jobs/", "^seek-oss/"],
+      "groupName": "buildkite plugins",
+      "managerBranchPrefix": "",
+      "matchManagers": ["buildkite"],
+      "recreateClosed": true,
+      "schedule": "after 3am and before 6am on Wednesday"
+    },
+    {
+      "commitMessageExtra": "",
+      "group": {
+        "commitMessageTopic": "{{groupName}}"
+      },
+      "groupName": "docker images",
+      "matchManagers": ["docker-compose", "dockerfile"],
+      "recreateClosed": true,
+      "schedule": "after 3am and before 6am on Wednesday"
+    },
+    {
+      "matchPackageNames": ["braid-design-system", "sku", "skuba"],
+      "matchPackagePatterns": ["^@?seek", "seek$"],
+      "prPriority": 98,
+      "schedule": "after 3am and before 6am on every weekday"
+    },
+    {
+      "commitMessageExtra": "",
+      "depTypeList": ["dependencies"],
+      "groupName": "aws-sdk",
+      "matchManagers": ["npm"],
+      "matchPackageNames": ["aws-sdk"],
+      "matchPackagePatterns": ["^@aws-sdk/"],
+      "recreateClosed": true,
+      "matchUpdateTypes": ["major"],
+      "schedule": "after 3am and before 6am on the first day of the month"
+    },
+    {
+      "automerge": true,
+      "depTypeList": ["devDependencies"],
+      "matchPackageNames": ["sku", "skuba"],
+      "matchUpdateTypes": ["major"]
+    },
+    {
+      "extends": ["group:monorepos"],
+      "excludePackageNames": [],
+      "recreateClosed": true
+    },
```

Comment:
Does this need a `matchUpdateTypes`? I'm not sure if it'll work with the `extends`, but it may be worth a shot.

Reply:
I'm guessing this didn't work?

Reply:
Do we even need the monorepos you recon?